src/ValidationModels/penaltyValidation.js

```diff
@@ -21,14 +23,14 @@ const trailerSchema = {
 
 const valueSchema = {
 	penaltyType: Joi.string().regex(/^(IM|CDN|FPN)$/).required(),
-	paymentStatus: Joi.string(), // .regex(/^(UNPAID|PAID)$/).required(),
-	paymentAuthCode: Joi.string(),
+	paymentStatus: Joi.string().valid(['PAID', 'UNPAID']),
+	paymentAuthCode: Joi.string().alphanum(),
 	// .when('paymentStatus', { is: 'PAID', then: Joi.required() }),
 	paymentDate: Joi.number().integer(),
 	// .when('paymentStatus', { is: 'PAID', then: Joi.required() }),
-	paymentToken: Joi.string().required(),
-	formNo: Joi.string(),
```

Comment:
`formNo` is gone, was it unused?

Reply:
Yes it's never assigned a value in the iOS app and none of the services use it.